scripts/provisioning/s3_setup

```diff
@@ -178,14 +173,22 @@ def create_logger():
 
     return logger
 
-def create_logger_directory():
-    """Create log directory if not exsists."""
-    _logger_directory = os.path.join(s3deployment_log_directory)
-    if not os.path.isdir(_logger_directory):
+def create_logger_directory(config_file):
+    """Create log directory if not exists."""
+    # Get the machine id from confstore
+    s3_confkeys_store = S3CortxConfStore(f'yaml:///opt/seagate/cortx/s3/mini-prov/s3_prov_config.yaml', 'machine_id_index')
+    s3_provkeys_store = S3CortxConfStore(f'yaml://{config_file}', 'config_log_index')
+
+    s3_prov_config_path = s3_provkeys_store.get_config('cortx>common>storage>log')
+    global s3deployment_log_file
```

Comment:
Do we really need this 'Global' ?

Reply:
![image](https://user-images.githubusercontent.com/68695688/134892744-59e83417-0830-4560-ae6f-ba54ab3d4baa.png)

Reply:
yes. somehow the s3_deployment_file value is not getting updated on create_logger function.